tests/test_linefill_dra.py

```
@@ -1684,3 +1700,137 @@ def fake_composite(
+    assert _profile_pairs(pm._segment_profile_from_queue(queue, 100.0, 80.0)) == [
+        (14.3, 3.0),
+        (65.7, 0.0),
+    ]
+    c_profile = pm._segment_profile_from_queue(queue, 180.0, 151.6)
+    assert c_profile[0] == pytest.approx((37.17, 0.0), rel=1e-6)
+    assert sum(length for length, ppm in c_profile if ppm > 0.0) == pytest.approx(114.3, rel=1e-6)
+
+    hourly_km = 7.0
+    flow_m3h = _volume_from_km(hourly_km, diameter)
+    queue_state = [{"length_km": length, "dra_ppm": ppm} for length, ppm in queue]
+    for hour in range(1, 4):
```

Ruff (0.15.12)

[warning] Loop control variable `hour` not used within loop body

Rename unused `hour` to `_hour`

(B007)